Source/Scene/MetadataSemantic.js

```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_BOUNDING_REGION: "TILE_BOUNDING_REGION",
+  /**
+   * A bounding sphere for a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[centerX, centerY, centerZ, radius]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_BOUNDING_SPHERE: "TILE_BOUNDING_SPHERE",
+  /**
+   * The minimum height of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>. This semantic is used to tighten bounding regions implicitly calculated <code>3DTILES_implicit_tiling</code>.
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_MINIMUM_HEIGHT: "TILE_MINIMUM_HEIGHT",
+  /**
+   * The maximum height of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>. This semantic is used to tighten bounding regions implicitly calculated <code>3DTILES_implicit_tiling</code>.
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_MAXIMUM_HEIGHT: "TILE_MINIMUM_HEIGHT",
+  /**
+   * A bounding box for the content of a tile, stored as an array of 12 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are the same format as for <code>boundingVolume.box</code> in 3D Tiles 1.0. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  CONTENT_BOUNDING_BOX: "CONTENT_BOUNDING_BOX",
+  /**
+   * A bounding region for the content of a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[west, south, east, north, minimumHeight, maximumHeight]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  CONTENT_BOUNDING_REGION: "CONTENT_BOUNDING_REGION",
+  /**
+   * A bounding sphere for the content of a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[centerX, centerY, centerZ, radius]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  CONTENT_BOUNDING_SPHERE: "CONTENT_BOUNDING_SPHERE",
+  /**
+   * The minimum height of the content of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  CONTENT_MINIMUM_HEIGHT: "CONTENT_MINIMUM_HEIGHT",
```

Comment:
Content min/max height are in the spec but not used in the implementation. Should we include them here or no?